hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/mapreduce/TestWALRecordReader.java

```diff
@@ -193,7 +193,8 @@ public void testWALRecordReader() throws Exception {
     WALEdit edit = new WALEdit();
     edit.add(new KeyValue(rowName, family, Bytes.toBytes("1"),
         System.currentTimeMillis(), value));
-    long txid = log.append(info, getWalKeyImpl(System.currentTimeMillis(), scopes), edit, true);
+    long txid =
+      log.append(info, getWalKeyImpl(System.currentTimeMillis(), scopes), edit, true, false);
```

Comment:
Would it make sense adding an override for append rather than make change everywhere append is called?